test/integration/command-logging-and-monitoring/command_logging_and_monitoring.prose.test.ts

```diff
@@ -0,0 +1,228 @@
+import { expect } from 'chai';
+
+import { DEFAULT_MAX_DOCUMENT_LENGTH, type Document } from '../../mongodb';
+
+describe('Command Logging and Monitoring Prose Tests', function () {
+  const loggerFeatureFlag = Symbol.for('@@mdb.enableMongoLogger');
+  const ELLIPSES_LENGTH = 3;
+  let client;
+  let writable;
+
+  context('When no custom truncation limit is provided', function () {
+    /*
+        1. Configure logging with a minimum severity level of "debug" for the "command" component. 
+           Do not explicitly configure the max document length.
```

Comment:
This line has trailing whitespace, and there are other lines as well, double check prettier has run on this file.

Reply:
Ah there are some trailing whitespaces, fixed them now, my linter didn't catch any of them originally unfortunately. Good catch.